prompt/prompt-executor/prompt-executor-clients/prompt-executor-ollama-client/src/commonMain/kotlin/ai/koog/prompt/executor/ollama/client/OllamaModelCardCache.kt

```diff
@@ -0,0 +1,108 @@
+package ai.koog.prompt.executor.ollama.client
+
+import ai.koog.prompt.executor.ollama.client.dto.*
+import io.github.oshai.kotlinlogging.KotlinLogging
+import io.ktor.client.*
+import io.ktor.client.call.*
+import io.ktor.client.request.*
+import io.ktor.http.*
+import kotlin.time.Clock
+import kotlin.time.Duration.Companion.minutes
+import kotlin.time.ExperimentalTime
+import kotlin.time.Instant
+
+/**
+ * Manages Ollama model card cache
+ */
+internal class OllamaModelCardCache(
+    private val client: HttpClient,
+    private val baseUrl: String,
+) {
```

Comment:
Do we really need the cache here?

Reply:
I guess we could get rid of that. I just kept it because it was there... :smile:
I don't mind removing it. Just tell me.

Reply:
Removed the cache, moved the code directly to the `OllamaClient` and further cleaned the HttpClient configuration.